rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -149,10 +154,21 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
                                                                                        toArray(policyParams),
                                                                                        toArray(policyFileParams));
 
-            if (jsonResult.equals("true")) {
-                jsonResult = createNonEscapedSimpleJsonPair("result", "true");
-            }
+            JSONValue json = JSONParser.parseStrict(jsonResult);
+            JSONObject obj = json.isObject();
+            if (obj != null) {
+                if (obj.containsKey("valid")) {
```

Comment:
You could have inserted the if in the first one since it the first only contains one if, but not very important